templates/cfp/new.html

```diff
@@ -96,30 +101,35 @@ <h2>Submit a {{form.model.human_type}} proposal</h2>
       {% endcall %}
 
       {% if cfp_type in ('workshop', 'youthworkshop') %}
+        <legend>Attendees</legend>
+
+          {% if cfp_type != 'youthworkshop' %}
+          <p>Our workshop tents can accommodate 10-50 attendees per session; if your workshop
+          is smaller than that, you don't need to use the Call for Participation, but feel free to
+          <a href="mailto:{{ config['CONTENT_EMAIL'][1] }}">drop us an email</a> instead.</p>
+          {% endif %}
+          <p>If volunteers would help you increase capacity, please say so.
```

Comment:
Nitpicky: I think there's a missing closing-paragraph tag here.

```suggestion
          <p>If volunteers would help you increase capacity, please say so.</p>
```